### ISSUE TITLE
Program crashed when open the website in Windows

### ISSUE THREAD

**Issue by LittleYang0531** [SonolusHaniwa/sonolus-server-cpp#1]
**Describe the bug**
Program crashed.

**To Reproduce**
1. Run the program
2. Open the browser and enter `http://127.0.0.1:8080/levels/create`
3. See error

**Expected behavior**
The program should response correctly.

**Desktop (please complete the following information):**
 - OS: Windows 11 Professional
 - Browser: Microsoft Edge
 - Version: 1.0.2

**Comment by LittleYang0531** [comment]
After testing, Linux systems are not affected, only Windows systems experience this problem.

**Comment by LittleYang0531** [comment]
gdb debug log:

```bash
(gdb) r
Starting program: C:\Users\littleyang\Desktop\sonolus-server-cpp\sonolus.exe
[New Thread 13604.0x33fc]
[New Thread 13604.0x3234]
[New Thread 13604.0x38e4]
warning: Can not parse XML library list; XML support was disabled at compile time
warning: onecore\net\netprofiles\service\src\nsp\dll\namespaceserviceprovider.cpp(550)\nlansp_c.dll!00007FF86455D93D: (caller: 00007FF87A82ACF6) LogHr(1) tid(37fc) 8007277C
[New Thread 13604.0x3988]
[New Thread 13604.0x1968]
[New Thread 13604.0x141c]
[New Thread 13604.0x3854]
[New Thread 13604.0x7c8]
[New Thread 13604.0x2798]
[New Thread 13604.0x1330]
[New Thread 13604.0x3634]
[2023-01-20 15:22:56] [INFO] [tid:14332] [httpd.h:972] Listening...
[2023-01-20 15:22:58] [INFO] [tid:1992] [httpd.h:1094] New Connection: GET /levels/create [127.0.0.1:15822]

Thread 9 received signal SIGSEGV, Segmentation fault.
[Switching to Thread 13604.0x7c8]
0x00007ff846d53f50 in ?? ()
(gdb) bt
#0  0x00007ff846d53f50 in ?? ()
Backtrace stopped: previous frame identical to this frame (corrupt stack?)
```

After searching online, it can be comfirmed that this error was caused by the stack pointer was broken.

**Comment by LittleYang0531** [comment]
After further debugging, the problem was found to be caused by `mysql_query()`, so it is left untouched for now.

**Comment by LittleYang0531** [comment]
Take back foreword, after further debugging, we found that the problem is already present in the child process after it is created, and the root cause of the problem has not been found yet, so there will be no solution for now.

**Comment by LittleYang0531** [comment]
This problem may be caused by the environmental differences between Windows and Linux. We made breakpoint debugging at the same location of the source code in the two environments, and there was no exception in the function call stack of Linux, while the above error occurred in Windows. Therefore, the solution to this problem needs to be further debugged before it can be obtained. During this period, Windows applications cannot run normally. This issue will be closed after the solution is reached. Sorry, Windows users :(

**Comment by LittleYang0531** [comment]
We have found the root of the problem, which is caused by MYSQL structs that are not thread-safe. We will fix this issue in the next release 1.0.2-2.
